drivers/overlay/overlay.go

```diff
@@ -2162,9 +2162,15 @@ func supportsDataOnlyLayersCached(home, runhome string) (bool, error) {
 // ApplyDiffWithDiffer applies the changes in the new layer using the specified function
 func (d *Driver) ApplyDiffWithDiffer(id, parent string, options *graphdriver.ApplyDiffWithDifferOpts, differ graphdriver.Differ) (output graphdriver.DriverWithDifferOutput, errRet error) {
 	var idMappings *idtools.IDMappings
+	forceMask := options.ForceMask
+
 	if options != nil {
```

Comment:
Either the first line can crash, or the second one is unnecessary.

Reply:
thanks, opened a PR: https://github.com/containers/storage/pull/1984